samples/sample-app.gradle

```diff
@@ -0,0 +1,33 @@
+Properties localProperties = new Properties()
```

Comment:
can we add the common `plugin` as well?

Reply:
I tried it too but received the following error. Looks like gradle doesn't allow `plugin` block in this file 😞 

```
Only Project and Settings build scripts can contain plugins {} blocks
```